README.md

```diff
@@ -230,11 +230,15 @@ You can configure your dashboard for Basic Authentication by adding usernames an
       "user":"user2",
       "pass":"pass"
     }
-  ]
+  ],
+  "options": {
```

Comment:
I was trying to leave it open for future changes. Would you suggest that I just bring the setting up, or something else?